### DISCUSSION TITLE
Disable outside pvp option in Player Indicators should allow the option to ignore disabling friend/clanmates

### DISCUSSION THREAD

**Discussion by IEarnSolo** [runelite/runelite#16160] [Ideas]
The "Disable outside pvp" option in Player Indicators was a great addition. However I think it should allow an option to ignore disabling friends and clanmates. I think the main intent people had for this was to enable all non-friends and non-clanmates when entering pvp areas. Not disable everything outside of pvp. It's a massive bummer that it has to be all or nothing right now. I love the option, but won't be using it because I like seeing my friend's and clanmate's indicators outside of pvp. I'm sure lots of others would agree.

If the checkbox was changed to a dropdown menu where you had options for what you wanted to disable outside pvp - All, Friends, Clan Members, Others etc. - then that would satisfy all cases.

Also, player's names still show up as colored in right-click windows with disable outside pvp option enabled and you are outside of pvp.

![image](https://user-images.githubusercontent.com/105403369/212522546-e2d6a55c-02b6-4290-84f5-73171c470da2.png)

**Comment by rsValliance** [comment]
Came here to request a similar thing. Talking with mates it would be ideal if "Disable outside PvP" was a dropdown that allowed "Only for Highlight Others" - "All Highlight Options" - "Off" or similar. 

So that way the existing functionality still exists, but we can turn it on for JUST highlighting others, as essentially just want this for Wildy prayer training/pvm and not to always hide friends and clanmates for the whole game.
